### ISSUE TITLE
Undefined offset calling getLabel()

### ISSUE THREAD

**Issue by dapphp** [Spomky-Labs/otphp#46]
In development with error reporting turned on, this is getting in the way of generating provisioning URIs and pulling the label out of them for Ajax responses.

Consider:

    $uri = 'otpauth://totp/My%20Test%20-%20Auth?secret=12345678901234';

    $otp = \OTPHP\Factory::loadFromProvisioningUri($uri);

    $label = $otp->getLabel();
    echo $label;

Produces:

```
PHP Notice:  Undefined offset: 1 in /tmp/otp/vendor/spomky-labs/otphp/src/Factory.php on line 103
PHP Stack trace:
PHP   1. {main}() /tmp/otp/test.php:0
PHP   2. OTPHP\Factory::loadFromProvisioningUri() /tmp/otp/test.php:16
PHP   3. OTPHP\Factory::createOTP() /tmp/otp/vendor/spomky-labs/otphp/src/Factory.php:31
PHP   4. OTPHP\Factory::getLabel() /tmp/otp/vendor/spomky-labs/otphp/src/Factory.php:88
PHP Notice:  Undefined offset: 1 in /tmp/otp/vendor/spomky-labs/otphp/src/Factory.php on line 56
PHP Stack trace:
PHP   1. {main}() /tmp/otp/test.php:0
PHP   2. OTPHP\Factory::loadFromProvisioningUri() /tmp/otp/test.php:16
PHP   3. OTPHP\Factory::populateOTP() /tmp/otp/vendor/spomky-labs/otphp/src/Factory.php:33
string(13) "My Test - Auth"
```

It appears if the URI doesn't contain both an issuer AND a label, this warning will be generated.

The same error is also emitted from the populateOTP method in Factor.php when the path only contains a label:

    list($issuer, $label) = explode(':', rawurldecode(mb_substr($data['path'], 1, null, '8bit')));

**Comment by Spomky** [comment]
Thanks for reporting me this issue.
I released the `v7.0.3` that should fix this bug. A test based on your example has been added.

Please could you update your project and verify it works as expected?
If you do not encounter this issue anymore, we can close this issue.

Regards.

**Comment by dapphp** [comment]
Thanks for the super quick fix.  I am no longer getting the warnings with the same code and I get the correct label as expected.  Looks good!

Also thank you for the great OTP library.  I checked out at least 5 and for various reasons thought this was by far the best, properly coded, and functional library for what we need.
